backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/salesOrder.feature

```diff
@@ -84,4 +84,75 @@ Feature: sales order
+      | ps_3       | pricing_system_name_67 | pricing_system_value_67 | pricing_system_description | true         |
+    And metasfresh contains M_PriceLists
+      | Identifier | M_PricingSystem_ID.Identifier | OPT.C_Country.CountryCode | C_Currency.ISO_Code | Name                    | OPT.Description | SOTrx | IsTaxIncluded | PricePrecision | OPT.IsActive |
+      | pl_67_1    | ps_3                          | DE                        | EUR                 | price_list_name_67_main | null            | true  | false         | 2              | true         |
+      | pl_67_2    | ps_3                          | DE                        | EUR                 | price_list_name_67_1    | null            | false | false         | 2              | true         |
+    And metasfresh contains M_PriceList_Versions
+      | Identifier | M_PriceList_ID.Identifier | Name                 | ValidFrom  |
+      | plv_67_1   | pl_67_1                   | salesOrder-PLV_67    | 2021-04-01 |
+      | plv_67_2   | pl_67_2                   | purchaseOrder-PLV_67 | 2021-04-01 |
+    And metasfresh contains M_ProductPrices
+      | Identifier | M_PriceList_Version_ID.Identifier | M_Product_ID.Identifier | PriceStd | C_UOM_ID.X12DE355 | C_TaxCategory_ID.InternalName |
+      | pp_67_1    | plv_67_1                          | p_3                     | 0.0      | PCE               | Normal                        |
+      | pp_67_2    | plv_67_1                          | p_33                    | 10.0     | PCE               | Normal                        |
+      | pp_67_4    | plv_67_2                          | p_31                    | 0.0      | PCE               | Normal                        |
+      | pp_67_5    | plv_67_2                          | p_32                    | 0.0      | PCE               | Normal                        |
+    And metasfresh contains PP_Product_BOMVersions:
+      | Identifier | M_Product_ID.Identifier | Name                |
+      | ppbv_67    | p_3                     | p_3_bomversion_name |
+    And metasfresh contains PP_Product_BOM:
+      | Identifier | M_Product_ID.Identifier | Name         | BOMType       | BOMUse        | C_UOM_ID.X12DE355 | PP_Product_BOMVersions_ID.Identifier | ValidFrom  |
+      | ppb_67     | p_3                     | p_3_bom_name | CurrentActive | Manufacturing | PCE               | ppbv_67                              | 2021-04-01 |
+    And metasfresh contains PP_Product_BOMLines:
+      | Identifier | PP_Product_BOM_ID.Identifier | M_Product_ID.Identifier | QtyBOM | C_UOM_ID.X12DE355 | ComponentType | ValidFrom  | Line |
+      | ppbl_67_1  | ppb_67                       | p_31                    | 3      | PCE               | CO            | 2021-04-01 | 100  |
+      | ppbl_67_2  | ppb_67                       | p_32                    | 4      | PCE               | CO            | 2021-04-01 | 200  |
```

Comment:
please create a sub-BOM for e.g. `p_31` and verify that the purchase order contains not `p_31`, but its components